src/components/PdfHighlighter.tsx

```diff
@@ -397,9 +397,10 @@ export const PdfHighlighter = ({
       if (highlightBindings?.container?.isConnected) {
         renderHighlightLayer(highlightBindings, pageNumber);
       } else {
-        const { textLayer } =
+        const { annotationLayer: textLayer } =
           viewerRef.current!.getPageView(pageNumber - 1) || {};
```

Comment:
This annotationLayer just so happened to be there - I don't know enough about PDF.js (yet) to know if this is a sensible place to put things.

It did have attribute `hidden` set so I made sure to unset that for the sake of this example.

Reply:
Turns out when the PDF has actual pdf-annotations this layer is also rotated, I ended up adding a dedicated sibling div instead